solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -80,16 +83,33 @@ public void call(ClusterState state, ZkNodeProps message, @SuppressWarnings({"ra
     split(state, message,(NamedList<Object>) results);
   }
 
+  /**
```

Comment:
Can we use HTML tags for the list? Otherwise the formatting will get messed up in editor tooltips and on the javadoc page.